app/src/main/java/com/keylesspalace/tusky/db/AppDatabase.java

```diff
@@ -674,4 +678,40 @@ public void migrate(@NonNull SupportSQLiteDatabase database) {
 
     @DeleteColumn(tableName = "AccountEntity", columnName = "activeNotifications")
     static class MIGRATION_49_50 implements AutoMigrationSpec { }
+
+    /**
+     * TabData.TRENDING was renamed to TabData.TRENDING_TAGS, and the text
+     * representation was changed from "Trending" to "TrendingTags".
+     */
+    public static final Migration MIGRATION_52_53 = new Migration(52, 53) {
+        @Override
+        public void migrate(@NonNull SupportSQLiteDatabase database) {
+            database.beginTransaction();
+            try {
+                Cursor cursor = database.query("SELECT id, tabPreferences FROM AccountEntity");
```

Comment:
Could this be a simple `UPDATE AccountEntity SET tabPreferences = replace('..Trending..', '...Tags..', tabPreferences` (or similar)?

Reply:
Yes. That would be much smarter way to do it. Done.